### PR TITLE
Move common transaction state/mutations into statelessTransaction 

### PR DESCRIPTION
**Pull request by c-johnson**
Cuts down on the state mutation boilerplate for executing transactions by moving all the common transaction details into statelessTransaction.js

### CODE REVIEW

**Comment by pkova** [comment]
I think you forgot (to commit) CreateGalaxy.js! 

Apologies if this is a WIP PR, noticed this since I'm doing some work on #93 but realized I better wait for this one to be merged.

File: src/bridge/views/AcceptTransfer.js [hunk 1]
```diff
@@ -32,144 +32,17 @@ class AcceptTransfer extends React.Component {
...
-            gasPrice: fromWei(r[2], 'gwei'),
-          };
-
-          this.setState({...txMetadata})
-
-        })
-      }
-    });
+    this.createUnsignedTxn = this.createUnsignedTxn.bind(this)
+    this.statelessRef = React.createRef();
```
Comment (Fang-) [inline src/bridge/views/AcceptTransfer.js]: The React docs say you need to add a `ref` attribute to some DOM element for this to work? I know this ends up with having a pointer to the `StatelessTransaction` in `statelessRef.current`, but how does that hook up? I don't see it happen anywhere.

Comment (c-johnson) [reply src/bridge/views/AcceptTransfer.js]: Ah, good catch! I missed this one. It's there now.

**Comment by c-johnson** [comment]
Indeed I did @pkova! It's updated now.

**Pull request merged**